homeassistant/components/modbus/const.py

```diff
@@ -68,6 +68,7 @@
 ATTR_ADDRESS = "address"
 ATTR_HUB = "hub"
 ATTR_UNIT = "unit"
+ATTR_SLAVE = "slave"
```

Comment:
We should avoid using this term, an use more neutral terms instead

<https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/m/master-slave>

Reply:
We use "slave" because the Modbus standard uses "slave", apart from that it has been used for a long time in the other parts of the configuration.

Reply:
🤷 Right